api/src/paths/reports/appreport/view.ts

```diff
@@ -0,0 +1,141 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SYSTEM_ROLE } from '../../../constants/roles';
+import { getDBConnection } from '../../../database/db';
+import { authorizeRequestHandler } from '../../../request-handlers/security/authorization';
+import { ReportService } from '../../../services/reports-service';
+import { getLogger } from '../../../utils/logger';
+
+const defaultLog = getLogger('paths/reports/appreport/view');
```

Comment:
could be renamed to something a bit more descriptive. ie: `paths/reports/app/view` maybe? appreport seems confusing

Reply:
it will change, for now I left like that